jobs/cluster-6/templates/bin/post-deploy

```diff
@@ -1,19 +1,46 @@
 #!/bin/bash
+
+function log() {
+  echo "[$(date)] $*"
+}
+
 <% if !spec.bootstrap %>log "this node is not the bootstrap node; nothing to do."
 exit 0
 <% else %>set -eu
 exec >&2
 
+<% if p('redis.tls.enabled') -%>
+# Install and setup stunnel.
+apt install -y redis-tools stunnel4
```

Comment:
These should be switched to packages with the redis boshrelease, happy to help with writing those packages scripts if it's helpful.

Reply:
I was suspecting this would be the case, we then need to embed the package in the redis forge.